cdf.toml

```
@@ -0,0 +1,32 @@
+[cdf]
+project_dir = "cognite_toolkit"
+
+[modules]
+# This is the version of the modules. It should not be changed manually.
+# It will be updated by the 'cdf module upgrade' command.
+version = "0.2.18"
+
+[modules.packages]
+data_model_examples = [
+    "cdf_auth_readwrite_all",
+    "cdf_oid_example_data",
+    "cdf_apm_simple_data_model"
+]
+cdf_demo_infield = [
+    "cdf_auth_readwrite_all",
+    "cdf_oid_example_data",
+    "cdf_apm_base",
+    "cdf_infield_common",
+    "cdf_infield_location",
+]
+cdf_infield = [
+    "cdf_auth_readwrite_all",
+    "cdf_apm_base",
+    "cdf_infield_common",
+    "cdf_infield_location",
+]
+example_pump = [
+    "example_pump_asset_hierarchy",
+    "cdf_asset_source_model",
+    "example_pump_data_model",
+]
```

I think this is the way forward. Looking at the interactive init and the [manifest files](https://github.com/cognitedata/toolkit/blob/main/cognite_toolkit/_cdf_tk/prototypes/_packages/quickstart/manifest.yaml), the only thing missing (?) is a more explanatory text for each module to print in the interaction. But that can be solved in other ways. How about this? 
```suggestion
data_model_examples = {
    title: "Data model examples to learn from", 
    cdf_auth_readwrite_all: "Access groups that shouldn't be here",
    cdf_oid_example_data: "Actual sample data from the Valhall platform",
    cdf_apm_simple_data_model: "This is out of fashion anyway"
},
cdf_demo_infield = {
    title: "End-to-end setup of InField demo",
    cdf_auth_readwrite_all: "redundant groups",
    cdf_oid_example_data: "Actual sample data from the Valhall asset",
    cdf_apm_base: ""
    cdf_infield_common: "",
    cdf_infield_location: "",
}

...```